buildlib/tools/check_tls_perf_caps.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -eE
+#
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See file LICENSE for terms.
+#
+
+old_out=$(${UCX_LEGACY_PATH}/bin/ucx_info -d)
+new_out=$(${UCX_PR_PATH}/bin/ucx_info -d)
+
+res=true
+for tl_name in `echo "${old_out}" | grep Transport | awk '{print $3}'`
```

Comment:
detail: maybe `$()` instead of `` to match others?

Reply:
fixed in place as pr is small